app/graphql/mutations/pairings/create_pairings.rb

```diff
@@ -0,0 +1,17 @@
+module Mutations
+  module Pairings
+    class CreatePairings < ::Mutations::BaseMutation
+      argument :pairer_id, Integer, required: true
+      argument :pairee_id, Integer, required: true
+      argument :date, String, required: true
+      argument :time, String, required: true
+      argument :notes, String, required: false
```

Comment:
Hate to throw a wrench in this, but I was double-checking this against `src/queries/index.js` and it takes a very different than we thought:
```ruby
mutation {
    createPairings(
      pairings: ${pairingsString}
    ) {
      pairer
      date
      time
    }
  }
```
I searched the project to find what "pairingsString" might actually look like, and found this in `src/mockData/index.js`:
```
mutation {
    createPairings(
      pairings: [{date:\"Mon Apr 29 2019\",time:\"morning\",pairer:\"5caa7eebfdebb8348e53a48e\"},{date:\"Mon May 06 2019\",time:\"morning\",pairer:\"5caa7eebfdebb8348e53a48e\"},{date:\"Mon May 13 2019\",time:\"morning\",pairer:\"5caa7eebfdebb8348e53a48e\"},{date:\"Mon May 20 2019\",time:\"morning\",pairer:\"5caa7eebfdebb8348e53a48e\"},{date:\"Mon May 27 2019\",time:\"morning\",pairer:\"5caa7eebfdebb8348e53a48e\"},{date:\"Mon Jun 03 2019\",time:\"morning\",pairer:\"5caa7eebfdebb8348e53a48e\"},{date:\"Mon Apr 29 2019\",time:\"afternoon\",pairer:\"5caa7eebfdebb8348e53a48e\"},{date:\"Mon May 06 2019\",time:\"afternoon\",pairer:\"5caa7eebfdebb8348e53a48e\"},{date:\"Mon May 13 2019\",time:\"afternoon\",pairer:\"5caa7eebfdebb8348e53a48e\"},{date:\"Mon May 20 2019\",time:\"afternoon\",pairer:\"5caa7eebfdebb8348e53a48e\"},{date:\"Mon May 27 2019\",time:\"afternoon\",pairer:\"5caa7eebfdebb8348e53a48e\"},{date:\"Mon Jun 03 2019\",time:\"afternoon\",pairer:\"5caa7eebfdebb8348e53a48e\"}]
    ) {
      pairer
      date
      time
    }
  }
```
Looks like the single argument is a JSON string; we'll need to parse it, then iterate over it to create a pairing for each object. So there are some changes we'll need to make through this PR to reflect that.